docs/en/index.md

```diff
@@ -15,6 +15,9 @@ News:
 <p>If you rely on the name of the “product admin group” (e.g., <code>_product_admin_&lt;product name&gt;</code>) you will also be impacted and have to update your scripts.</p>
 <p>You should avoid any logic that expects fixed group names as these are liable to change without notice. We recommend using the Get Groups and Profiles API to fetch the latest group information.</p>
 <hr class="api-ref-rule">
+<p>Since May 6th, 2023, UMAPI supports OAuth Server-to-Server workflows; JWT one is deprecated and it will be removed on 1st of January 2025, but existing integrations based on this authorization scheme will continue to work as usual until that date. Migrate your project to use OAuth instead before 2025. 
```

Comment:
```suggestion
<p>Since May 6th, 2023, UMAPI supports OAuth Server-to-Server (S2S) workflows; The JWT one is deprecated and will be removed on 1st of January 2025, but existing integrations based on this authorization scheme will continue to work as usual until that date. Migrate your project to use OAuth S2S instead before 2025. 
```